cmd/kgo-verifier/main.go

```diff
@@ -60,6 +61,7 @@ func makeWorkerConfig() worker.WorkerConfig {
 		BatchMaxbytes:      uint(*batchMaxBytes),
 		SaslUser:           *username,
 		SaslPass:           *password,
+		Name:               *name,
```

Comment:
Is this safe to set when using the consumer group reader?  I vaguely recall that when I added the Name field to the worker config I avoided setting it from there because the consumer group reader would get confused.

Reply:
i think that client id is independent from group member id, but will verify that